specification/resources/resource-manager/readme.md

```diff
@@ -57,12 +57,22 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-deploymentscripts-2019-10-preview
+tag: package-2019-11
 ```
 
+
+### Tag: package-2019-11
+
+These settings apply only when `--tag=package-2019-11` is specified on the command line.
+
+```yaml $(tag) == 'package-2019-11'
+input-file:
+  - Microsoft.Resources/stable/2019-11-01/subscriptions.json
```

Comment:
Isn't this against the convention? Shouldn't it be `package-subscriptions-2019-11`?